distance.go

```diff
@@ -0,0 +1,4675 @@
+package tlsh
+
+// bitPairsDiffTable is pre-calculated table that represents approximation 
```

Comment:
Paper: https://github.com/trendmicro/tlsh/blob/master/TLSH_CTC_final.pdf

and use the source luke :)
https://github.com/trendmicro/tlsh/blob/master/src/tlsh_util.cpp#L84-L4694
https://github.com/trendmicro/tlsh/blob/master/src/gen_arr2.cpp#L1-L91